src/mapping/RadialGeoMultiscaleMapping.cpp

```diff
@@ -0,0 +1,207 @@
+#include "RadialGeoMultiscaleMapping.hpp"
+#include <Eigen/src/Core/Matrix.h>
+#include <algorithm>
+#include "mesh/Mesh.hpp"
+
+namespace precice::mapping {
+
+RadialGeoMultiscaleMapping::RadialGeoMultiscaleMapping(
+    Constraint     constraint,
+    int            dimensions,
+    MultiscaleType type,
+    MultiscaleAxis axis)
+    : Mapping(constraint, dimensions, false, Mapping::InitialGuessRequirement::None),
+      _type(type), _axis(axis)
+{
+  setInputRequirement(Mapping::MeshRequirement::VERTEX);
+  setOutputRequirement(Mapping::MeshRequirement::VERTEX);
+}
+
+void RadialGeoMultiscaleMapping::computeMapping()
+{
+  PRECICE_TRACE(output()->vertices().size());
+
+  PRECICE_ASSERT(input().get() != nullptr);
+  PRECICE_ASSERT(output().get() != nullptr);
+
+  size_t const inSize  = input()->vertices().size();
+  size_t const outSize = output()->vertices().size();
+
+  int effectiveCoordinate = _axis;
+  PRECICE_ASSERT(effectiveCoordinate == 0 ||
+                     effectiveCoordinate == 1 ||
+                     effectiveCoordinate == 2,
```

Comment:
In this case, the hardcoded values are the respective enums because they lack an initializer. 
"If the first enumerator does not have an initializer, the associated value is zero. For any other enumerator whose definition does not have an initializer, the associated value is the value of the previous enumerator plus one." (from https://en.cppreference.com/w/cpp/language/enum)

Reply:
Then, as long as this already works, we can keep it like this for now. That's details that does not block the merge.

Reply:
Replaced with the enums (`MultiscaleAxis::X` etc) and set the values of the enum explicitly to 0,1,2, for clarity.